spec/src/main/asciidoc/microprofile-opentracing.asciidoc

```diff
@@ -144,6 +144,10 @@ Spans created for incoming requests will have the following tags added by defaul
 If there is an exception object available the implementation SHOULD also add logs `event=error` and `error.object=<error object instance>` to the active span.
 
 ==== Span creation and injection for outbound requests
+Tracing in `javax.ws.rs.Client` has to be explicitly enabled by invoking
+`org.eclipse.microprofile.opentracing.ClientTracingRegistrar.configure(ClientBuilder clientBuilder)`.
+Implementation might enable client tracing globally, then explicit configuration has no effect.
```

Comment:
The implementation ...  , in this case explicit ...

Reply:
@pilhuhn thanks, text has been changed